docs/installation.md

```diff
@@ -0,0 +1,44 @@
+# Installing Pop!_OS
+
+The Pop!_OS installer guides the user through setup of basic system settings:
+
+- Locale and keyboard language
+- Hard drive selection and partitioning (advanced)
+- User account creation
+- System disk encryption
+
+## Installation
+
+1. Choose a language. 
+
+![Choose Language](images/installation/choose-language.png)
+
+2. Select a keyboard language layout.
+
+![Keyboard Language](images/installation/keyboard-language.png)
+
+3. Choose a clean or custom installation. <!-- See Using [GParted Custom (Advanced)](advanced-installation.md) options. -->
+
+![Clean Install](images/installation/clean-install.png)
+
+4. Choose the target drive for the OS installation. Click Erase and Install.
+
+![Select Drive](images/installation/select-system-drive.png)
+
+5. Enter a username.
```

Comment:
This is just an idea, but it might be useful to call out that it's helpful for the user to remember their username in case they want to perform certain operations later. Don't phrase it like it's something the user _needs_ to do, just something like:

> **Tip:** You don't need to type your username when logging into Pop!_OS normally, but it can be handy to remember when logging into a terminal session, logging in remotely via SSH, or performing certain configurations.

Separate but related, this step should be reworded to reflect that the user should enter their full name in the Full Name input box, and (if you want to get into the details) explain that the system will suggest a username based on the full name. You can also include the username restrictions if you'd like.

The example full name in the screenshot should be formatted as a full name with capital letters and a space (e.g. `Pop User` instead of `pop-user`) to serve as an example of what the user can expect from a suggested username vs. their full name.

Reply:
I can do this, but for privacy reasons I personally never tie my actual full name to an OS. Asking for a full name may be off-putting for some. What would you think of "Enter a full name" instead of "Enter _your_ full name"?

Reply:
Frankly, I think that's a little silly and would result in more awkward wording than is necessary. We want people to feel comfortable providing their full name into the AccountsService metadata of their Pop!_OS machine, and if they don't, they can absolutely choose to enter a fake name or anything else.

If you feel very strongly about this, it could be something like "Enter your full name (or the display name you wish to use)".

Reply:
Changed to "Enter your first and last name" in the latest commit.

This is out of my depth, but I know browsers do a lot of fingerprinting based on information provided by the system. Is it possible for the user name or full name to be a part of that information?

Reply:
After double-checking, full name isn't even stored in AccountsService, it's stored in `/etc/passwd`, even more standard for Linux and some UNIX systems. It's definitely local, it's a text file.

That said, you can see with `ls -alh /etc/passwd` that it is world-readable, which means everyone with access to the system can see the list of users with their full names (they'd have to use another method to find out who's currently logged in or running an application.) You can see your own line with `getent passwd $(whoami)`, which means processes running under your user can also see that.

And it does look like at least [Chrome accesses /etc/passwd sometimes](https://www.reddit.com/r/linux/comments/3iuogc/chrome_does_access_your_etcpasswd_file_selinux/), and looking at the Chromium source code, that [does include the full name](https://source.chromium.org/search?q=pw_gecos&sq=&ss=chromium) (called `pw_gecos` by the `getpwnam` function that they use.)

So yes, your web browser can have access to your full name if the web browser wants to, and it is theoretically possible for it to pass that information on to a website. Now, doing some searching for "javascript get user's full name", "javascript read /etc/passwd", "read /etc/passwd through web browser", etc, I don't see any indication that browsers would currently choose to share that information with a website. There is no standard or mechanism that would cause them to do that intentionally. But it is certainly possible that e.g. a browser vulnerability that allows accessing arbitrary files could read `/etc/passwd`.

I didn't mean to shoot the concern down entirely; thinking about it more, I can see why you'd want to make that distinction (where e.g. iCloud or Microsoft accounts might require real names in their terms of service.) Like I said, mentioning a generic "display name" as a secondary option would be perfectly fine. "A full name" would just be a weirdly vague way to talk about the user's full name.

Reply:
fyi, last I heard we are moving the docs to a separate repo(pop-os/docs) and subdomain(docs.pop.os.org), as mdbooks does not play nice with being a subfolder of nuxt.

Reply:
@jacobgkau Thanks for digging into that.